integration_tests/test_fixtures.go

```diff
@@ -0,0 +1,52 @@
+package integration_tests
+
+import (
+	"github.com/FleekHQ/space-daemon/config"
+	spaceEnv "github.com/FleekHQ/space-daemon/core/env"
+	"github.com/namsral/flag"
```

Comment:
Not I'm using `https://github.com/namsral/flag` here instead of the std go `flag` module. They have similar interfaces, it's just that this flag module also checks the environment for keys similar to the uppercase version of flag names.